server/readme.md

```diff
@@ -0,0 +1,79 @@
+# Payment Gateway Integration using Yoco API
+
+This is a simple payment gateway integration example using the Yoco API. It is built using Node.js and Express.js.
+
+## Getting Started
+
+### Prerequisites
+
+To use this code, you will need the following:
+
+- Node.js (v12 or higher)
+- Yoco account
+- Secret Key (from Yoco dashboard)
+
+### Installation
+
+1.  Clone the repository
+2.  Install dependencies using `npm install`
+3.  Create a `.env` file in the root directory with the following information:
+
+`SECRET_KEY=<your_secret_key>`
+
+### Usage
+
+This code exposes a single endpoint `/` that accepts a POST request. It expects the following parameters in the request body:
+
+- `token`: A token generated by the Yoco SDK
+- `amountInCents`: The amount to charge in cents
+- `currency`: The currency to use (e.g., `ZAR`, `USD`, etc.)
+
+To use this code, make a POST request to the endpoint `/` with the required parameters in the request body.
+
+### Example
+
+javascriptCopy code
+
+    `const axios = require("axios");
+    import { Router } from "express";
```

Comment:
You can easily link to the documentation of yoco since they outline this on the Charge api